scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.43.0] - 2020-06-15
```

Comment:
I think this is more of a patch version (1.42.4)

Reply:
Ok